post.css

```
@@ -130,13 +130,15 @@ select {
 
 .second_section {
     margin-top: 24px;
-    padding-left: 20px;
+   
+}
+.second_section_block{
+     padding-left: 20px;
```

identation is off here
```suggestion
  padding-left: 20px;
```